aidants_connect_web/templates/aidants_connect_web/espace_responsable/home.html

```diff
@@ -0,0 +1,124 @@
+{% extends 'aidants_connect_web/espace_responsable/layout/main.html' %}
+
+{% load static %}
+
+{% block title %}Espace Référent - Aidants Connect{% endblock %}
+
+{% block extracss %}
+  <link href="{% static 'css/espace-aidant.css' %}" rel="stylesheet">
+{% endblock extracss %}
+
+{% block content %}
+  <div class="espace-referent">
+    <div class="fr-container-fluid fr-background-contrast--pink-tuile fr-py-12w">
+      {% include "layouts/_messages.html" %}
+      <div class="fr-container">
+        <h1>Bienvenue {{ referent.get_full_name }} !</h1>
+        <div class="fr-grid-row fr-grid-row--gutters">
+          <div class=" fr-col-md-3 fr-col-12">
+            <div class="fr-tile fr-tile--sm fr-tile--horizontal fr-enlarge-link" >
+              <div class="fr-tile__body">
+                <div class="fr-tile__content">
+                  <h3 class="fr-tile__title">
+                    <a href="{% url 'espace_responsable_aidants' %}">Gérer les comptes des aidants</a>
+                  </h3>
+                </div>
+              </div>
+              <div class="fr-tile__header">
+                <div class="fr-tile__pictogramm">
+                  <img class="fr-responsive-img" src="{% static 'images/icons/success.svg' %}" alt="" />
```

Comment:
Ces icônes sont déjà embarquées dans [`django-dsfr`](https://github.com/numerique-gouv/django-dsfr/tree/main/dsfr/static/dsfr/dist/artwork/pictograms).

```suggestion
                  <img class="fr-responsive-img" src="{% static 'dsfr/dist/artwork/pictograms/system/success.svg' %}" alt="" />
```